apps/docs/src/guide/predicates/methods.md

```diff
@@ -0,0 +1,55 @@
+# Interacting With Predicates
+
+The `Predicate` class extends the [`Account`](https://docs.fuel.network/docs/fuels-ts/account/) class, inheriting all its methods. Therefore, there are multiple ways to interact with predicates, but broadly speaking, we can think about three:
+ - `Checking Balances`
+ - `Transactions`
+ - `Transfers`
+
+
+## Checking Balances
+
+### `getBalances`
+
+This will return the balances of all assets owned by the predicate.
+
+See also: [Checking Wallet Balances](https://docs.fuel.network/docs/fuels-ts/wallets/checking-balances/#getting-a-wallets-balance)
+
+### `getResourcesToSpend`
+
+This will return the resources owned by a predicate so that they can be added to a transaction request.
+
+This method is called under the hood when using [`transfer`](./methods.md#transfer) or [`createTransfer`](./methods.md#createtransfer).
+
+You may want to use this method when using a predicate in an existing transaction request.
+
+<<< @/../../docs-snippets/src/guide/predicates/interacting-with-predicates.test.ts#interacting-with-predicates-1{ts:line-numbers}
+
```

Comment:
Then the "see also" would go below this snippet here.

However, considering there is no `getBalances` for `Predicates`, why should we have this here?

```suggestion

See also:
 - [Checking Wallet Balances](https://docs.fuel.network/docs/fuels-ts/wallets/checking-balances/#getting-a-wallets-balance)
```

Reply:
Never mind.

Since `Predicates` extends from `Account`, there is also a `getBalances` for it (thanks, @Torres-ssf, for the heads-up).

With that in mind, forget everything I said. 🤦‍♂️ You could leave it as is or roll it back to your first version, as that would be more complete and remove the need for users to click around.

Perhaps the phrasing should target `Account` instead of `Wallets`, though. I'm unsure if that makes sense, but it can also be a follow-up. Sorry for the confusion.

Reply:
No worries, it was also a bit misleading for me before I understood that a predicate is a type of account at first. I think the current state makes sense and still references another example of getting balances that consumers may also be familiar with.